draft-ietf-scitt-architecture.md

```diff
@@ -996,6 +996,9 @@ See {{-RECEIPTS}} for the generic security considerations that apply to Verifiab
 ### Key Compromise
 
 It is important for Issuers and Transparency Services to clearly communicate when keys are compromised, so that Signed Statements can be rejected by Transparency Services or Receipts can be ignored by Relying Parties.
+Transparency Services whose keys have been compromised can roll back their Statement Sequence to its most recent known valid prefix, establish new credentials, and use them to issue fresh Receipts going forward.
+In cases where Issuer identity remains stable across credential rotation, Issuers can produce fresh statements for their existing Subjects signed with new credentials, and containing an incremented security version number.
```

Comment:
What is a "security version number" ?

Reply:
It's a number you rev up every time the security properties of something changes, let me find a good reference.